lib/hil/hil.yml

```diff
@@ -0,0 +1 @@
+url: http://127.0.0.1:5000
```

Comment:
I'm confused. I thought commit messages are very short.

Should I put this explanation into PR comment?